book/website/communication/social-media/social-media-twitter-tips.md

```diff
@@ -1,20 +1,20 @@
 (cm-social-media-twitter-tips)=
-# Tips for Starting with Twitter
+# Tips for Starting with X
 
-Twitter has 353 million monthly active users.
-You are limited to 280 characters and the style is more informal and chatty than other platforms like Linkedin.
-You can use links, photos (1200 x 628 pixels) and videos (max 2 minutes 20 secs) in similar ways to the other social media platforms.
+X (formerly Twitter) has 353 million monthly active users.
```

Comment:
I wonder if we should delete this? Number of active users has fallen dramatically and this figure is almost definitely out of date now

Reply:
Thanks for highlighting this one too. I'll update (or remove) this based on what's decided with the other case.

Reply:
I've removed this one. On reflection, I agree it's not providing much benefit here.